plugins/orchestrator/README.md

```diff
@@ -225,7 +225,7 @@ The plugin provides OpenAPI `v2` endpoints definition to facilitate communicatio
 In addition, by leveraging on OpenAPI spec, it is possible to generate clients and create CI steps.
 
 OpenAPI specification [file](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/openapi/openapi.yaml) is available in [orchestrator-common](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common).  
-OpenAPI specification documentation is available [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs/markdown/README.md)
+OpenAPI specification documentation is available [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs)
```

Comment:
This reverts a previous PR

Reply:
This depends on where we decide to put the new generated docs. Maybe we can add both links (this README is available in the website) so the user can select the format they prefer.

Reply:
this way the user goes to the folder in github and choose the format he wants

Reply:
But if you generate docs in different format in the same folder is a mess,, that's why I would have a folder for each format.

Reply:
i changed it to 2 dircetories